homeassistant/components/enocean/dongle.py

```diff
@@ -0,0 +1,66 @@
+"""Representation of an EnOcean dongle."""
+import glob
+import logging
+from os.path import basename, normpath
+
+from enocean.communicators import SerialCommunicator
+from enocean.protocol.packet import RadioPacket
+
+from homeassistant.components.enocean.const import (
+    LOGGER,
+    SIGNAL_RECEIVE_MESSAGE,
+    SIGNAL_SEND_MESSAGE,
+)
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class EnOceanDongle:
+    """Representation of an EnOcean dongle.
+
+    The dongle is responsible for receiving the ENOcean frames,
+    creating devices if needed, and dispatching messages to platforms.
+    """
+
+    @classmethod
+    def detect(cls):
```

Comment:
We're not using the `cls`. Maybe make it a static method instead?

Reply:
Indeed. I changed it to a static method.